app/models/like.rb

```diff
@@ -5,10 +5,6 @@ class Like < ApplicationRecord
   validates :author, presence: true
   validates :post, presence: true
 
-  # def update_likes_counter(_post_id)
-  #   post.increment!(:likes_counter)
-  # end
-
   after_save :update_like_counter
 
```

Comment:
- [ **OPTIONAL** ] `config/routes.rb` file: As right now you are you are using only two actions in your controller and to not have unnecessery routes, I can suggest you update you routes with the only used action. Later you can add other routes when needed. i.e.
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```